src/components/currency/Currency.js

```diff
@@ -7,24 +7,20 @@ import {NumberComponent} from '../number/Number';
 export class CurrencyComponent extends NumberComponent {
   constructor(component, options, data) {
     super(component, options, data);
-
     this.currency = this.component.currency || 'USD';
-    this.decimalLimit = this.component.decimalLimit || 2;
-
+    this.decimalLimit = options.decimalLimit || this.component.decimalLimit || 2;
     // Get the prefix and suffix from the localized string.
-    const regex = `(.*)?100(${this.decimalSeparator === '.'
-      ? '\\.'
-      : this.decimalSeparator}0{${this.decimalLimit}})?(.*)?`;
-    const parts = (100).toLocaleString(options.language, this.getFormatOptions()).match(new RegExp(regex));
+    const regex = `(.*)?100(${this.decimalSeparator}0{${this.decimalLimit}})?(.*)?`;
+    const parts = (100).toLocaleString(this.getBrowserLanguage(), this.getFormatOptions()).replace('.', this.decimalSeparator).match(new RegExp(regex));
     this.prefix = parts[1] || '';
-    this.suffix = parts[3] || '';
+      this.suffix = parts[3] || '';
   }
 
   getFormatOptions() {
     return {
       style: 'currency',
       currency: this.currency,
-      useGrouping: true,
+      useGrouping: false,
```

Comment:
Why are we changing this? Don't we want this to be true?

Reply:
That was testing purpose. I will fixed it right now.